bundle/edu.gemini.pot/src/test/scala/edu/gemini/spModel/target/SpTargetPioSpec.scala

```diff
@@ -24,28 +18,6 @@ import edu.gemini.shared.util.immutable.ScalaConverters._
 object SpTargetPioSpec extends Specification with ScalaCheck with Arbitraries {
 
   {
-    implicit val arbDistribution = Arbitrary[SpectralDistribution] {
-      Gen.oneOf(
-        BlackBody(8000),
-        BlackBody(10000),
-        PowerLaw(0),
-        PowerLaw(1),
-        EmissionLine(450.nm, 150.kps, 13.ergsPerSecondPerSquareCentimeter, 22.wattsPerSquareMeterPerMicron),
-        EmissionLine(550.nm, 400.kps, 23.wattsPerSquareMeter, 42.ergsPerSecondPerSquareCentimeterPerAngstrom),
-        LibraryStar.A0V,
-        LibraryStar.A5III,
-        LibraryNonStar.NGC2023,
-        LibraryNonStar.GammaDra
-      )
-    }
-    implicit val arbProfile = Arbitrary[SpatialProfile] {
-      Gen.oneOf(
-        PointSource,
-        UniformSource,
-        GaussianSource(0.5),
-        GaussianSource(0.75)
-      )
-    }
```

Comment:
These were moved to `edu.gemini.spModel.core.Arbitraries` since they're needed for core unit tests now.